src/main/resources/edu/cornell/kfs/krad/config/MaintainableXMLUpgradeRules.xml

```diff
@@ -0,0 +1,724 @@
+                <replacement>org.kuali.rice.kim.impl.identity.PersonImpl</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.kfs.fp.businessobject.WireCharge</match>
+                <replacement>org.kuali.kfs.sys.businessobject.WireCharge</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.kfs.fp.businessobject.DisbursementVoucherDocumentationLocation</match>
+                <replacement>org.kuali.kfs.sys.businessobject.PaymentDocumentationLocation</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.rice.kns.bo.StateImpl</match>
+                <replacement>org.kuali.rice.location.impl.state.StateBo</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.rice.kim.bo.impl.ReviewResponsibility</match>
+                <replacement>org.kuali.rice.kim.impl.responsibility.ReviewResponsibilityBo</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.rice.kns.bo.Note</match>
+                <replacement>org.kuali.kfs.krad.bo.Note</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.rice.krad.bo.Note</match>
+                <replacement>org.kuali.kfs.krad.bo.Note</replacement>
+            </pattern>
+            <pattern>
+                <match>org.kuali.rice.krad.bo.AdHocRoutePerson</match>
+                <replacement>org.kuali.kfs.krad.bo.AdHocRoutePerson</replacement>
+            </pattern>
+            <pattern>
+                <match>boNotes</match>
+                <replacement>(CONVERT_LEGACY_NOTES)</replacement>
+            </pattern>
+            <pattern>
+                <match>unserializable-parents</match>
+                <replacement></replacement>
+            </pattern>
+            <pattern>
+                <match>autoIncrementSet</match>
+                <replacement></replacement>
+            </pattern>
+            <pattern>
+                <match>map</match>
+                <replacement>(MOVE_NODES_TO_PARENT)</replacement>
+            </pattern>
+        </pattern>
+        <!-- KDO-873 -->
+        <pattern>
+            <class>organizationOwnerAccount</class>
+            <pattern>
+                <match>org.kuali.rice.kim.bo.impl.PersonImpl</match>
+                <replacement>org.kuali.rice.kim.impl.identity.PersonImpl</replacement>
+            </pattern>
+            <pattern>
+                <match>indirectCostRcvyFinCoaCode</match>
+                <replacement></replacement>
+            </pattern>
+            <pattern>
+                <match>indirectCostRecoveryAcctNbr</match>
+                <replacement></replacement>
+            </pattern>
+        </pattern>
+        <!-- KDO-873 end -->
+        <!-- KDO-934 -->
+        <!--
+            NOTE: We will need to revisit these VendorHeader conversion rules with the next
```

Comment:
We still have a vendorLocale property that's exclusive to our vendor header extension objects, but it appears to be getting removed by the rules below (due to current limitations of our XML conversion service customization). Is that presenting an issue with older docs? Or is there a user story to analyze this particular case further?